README.md

```diff
@@ -95,6 +96,18 @@ For a full command reference, see the list below, or visit [cli.netlify.com](htt
 
 Run any Netlify API method
 
+### [blobs](/docs/commands/blobs.md)
+
+(Beta) Manage objects in Netlify Blobs
+
+| Subcommand | description  |
+|:--------------------------- |:-----|
+| [`blobs:delete`](/docs/commands/blobs.md#blobsdelete) | (Beta) Deletes an object with a given key from a Netlify Blobs, if one exists  |
+| [`blobs:get`](/docs/commands/blobs.md#blobsget) | (Beta) Reads an object with a given key from a Netlify Blobs store and, if it exists, prints the content to the terminal or saves it to a file  |
+| [`blobs:list`](/docs/commands/blobs.md#blobslist) | (Beta) Reads an object with a given key from a Netlify Blobs store and, if it exists, prints the content to the terminal or saves it to a file  |
```

Comment:
[boulder] -  description is off here too ... I'm realizing now that this file might be autogenerated but I'm leaving this comment anyway